.github/workflows/dusk_ci.yml

```diff
@@ -12,7 +12,6 @@ jobs:
         with:
           profile: minimal
           toolchain: nightly
```

Comment:
Remove also all the `toolchain` since they're useless.

Reply:
Fixed